selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java

```diff
@@ -155,6 +156,7 @@ public void checkFindUsagesBaseOperation() {
     findUsages.selectNodeInFindUsagesPanel("AppController");
     findUsages.sendCommandByKeyboardInFindUsagesPanel(ENTER.toString());
     findUsages.waitExpectedTextIsNotPresentInFindUsagesPanel(EXPECTED_TEXT_2);
+    findUsages.selectNodeInFindUsagesPanel("AppController");
```

Comment:
Looks like there is separate steps started from `findUsages.selectNodeInFindUsagesPanel("AppController");`.
It's not quite clear what second call is for. So, it would be better to divide two steps by empty row and comment on it.